pip/compat.py

```diff
@@ -38,10 +39,7 @@
 
 if sys.version_info >= (3,):
     def console_to_str(s):
-        try:
-            return s.decode(sys.__stdout__.encoding)
-        except UnicodeDecodeError:
-            return s.decode('utf_8')
+        return s.decode(locale.getpreferredencoding() or "utf-8", 'replace')
```

Comment:
Nice catch also is this line less than 79 characters because of PEP8. I know I hate PEP8 sometimes myself as it sometimes gets annoying but after a while I realize that the reasons why it is annoying is to make it not only easy for my to patch / fix but for others to read.